### PR TITLE
Rename fixtures to integration tests

### PR DESCRIPTION
**Pull request by mrm007**
Also renamed a couple of jobs/steps ~~because I couldn't help myself~~ for consistency. I'll update branch protection with the new job name before merging.

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -44,16 +44,15 @@ jobs:
       - name: Install Dependencies
         run: pnpm i
 
-      - name: Build
-        run: pnpm build
-
       - name: Lint
         run: pnpm lint
```
Comment (askoufis) [inline .github/workflows/validate.yml:48]: I thought lint depended on build because we want to lint generated files, e.g. icons?

Comment (mrm007) [reply .github/workflows/validate.yml:48]: I don't remember that. But if lint depends on build, the relationship should be defined in the task graph.

Comment (askoufis) [reply .github/workflows/validate.yml:48]: Yeah good point. So the cache should handle it and not rebuild. 👍

File: packages/braid-design-system/codemod/bin.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env node
-require('./dist/index');
+require('./dist/index'); // eslint-disable-line import/no-unresolved
```
Comment (askoufis) [inline packages/braid-design-system/codemod/bin.js:2]: What changed the necessitated adding this?

Comment (mrm007) [reply packages/braid-design-system/codemod/bin.js:2]: Before the change, it was running build before lint

**Review by askoufis — APPROVED** [review]

**Pull request merged**